### ISSUE TITLE
Bug with relative paths

### ISSUE THREAD

**Issue by marcelhadorn** [Meteor-Community-Packages/meteor-scss#253]
The package has a mature flaw, by not respecting relative paths correctly.
I have components that have a folder structure similar to this:
```
main.scss
components
|── blockquote
|   |── _style.scss
|   └── _variables.scss
|── button
|   |── _style.scss
|   └── _variables.scss
└── ...
```
While this is the main.scss content:
```
@import 'components/blockquote/style';
@import 'components/button/style';
...
```

and each _style.scss has the following import:
```
@import 'variables';
```
which are relative paths.

However fourseven:scss will only respect the first relative path and then "assumes" that the `@import 'variables';` in `components/button/_style.scss` is the same as in `components/blockquote/_style.scss`, which obviously is wrong behaviour.

The app then crashes, because it is missing the variables from `components/button/_variables.scss`. If I place those variables in the first `variables.scss` the variables are found.

**Comment by sebakerckhof** [comment]
Afaik, this is because of limitations on the information we get from `node-sass`, but I'm happy to be proven wrong by a PR.

**Comment by marcelhadorn** [comment]
@sebakerckhof I'm not sure what you mean by "a PR"? Should we report this to `node-sass`, or what do you suggest? Am not familiar with this, usually using this file structure within a framework where we compiles scss with `gulp-sass`.

**Comment by sebakerckhof** [comment]
Yes, I meant Pull Request. It's already reported to node-sass, but there's been no indication they're interested in solving this for over a year.

**Comment by vuhrmeister** [comment]
I'm facing the same problem with Material Components for Web. Can't use it without having that issue fixed. Or any other idea how to work around this issue?

**Comment by vuhrmeister** [comment]
Since I now need this working now, I played around a bit with the code and I got it working so far. 
Will come back when I have something final.

**Comment by vinvinod** [comment]
@vuhrmeister Can you share the solution you found?

**Comment by sebastian-ruiz** [comment]
I am having the same problem. If I have a `_variables.scss` file in my project, and I also try and use font awesome with:
```
@import "{}/imports/packages/font-awesome/scss/font-awesome";
```
It breaks with error:
```
   While processing files with fourseven:scss (for target web.browser):
   /client/main.scss: Scss compiler error: Error: Undefined variable: "$fa-font-path".
   on line 6 of path
   from line 8 of {}/imports/packages/font-awesome/scss/font-awesome
   from line 8 of {}/client/main.scss
   >> /fontawesome-webfont.eot?v=#{$fa-version}');
   
   ------------------------------------------^
   
   
   error: Scss compiler error: Error: Undefined variable: "$fa-font-path".
   on line 6 of path
   from line 8 of {}/imports/packages/font-awesome/scss/font-awesome
   from line 8 of {}/client/main.scss
   >> /fontawesome-webfont.eot?v=#{$fa-version}');
   
   ------------------------------------------^
```
If I remove my local `_variables.scss` to something else, the error goes away.

**Comment by sebakerckhof** [comment]
Fixed per 4.14.1
